.travis.yml

```diff
@@ -9,6 +9,7 @@ branches:
   except:
     - piptools-ignore-patch
 install:
+  - "python -m pip install pip --upgrade"
```

Comment:
@jgonggrijp + to what've said using just `pip` won't fix the issue. As on Ubuntu, pip => pip for python2 and pip3 => pip for python3 so both should be upgraded ;)